pkg/proxy/ipvs/proxier_test.go

```diff
@@ -107,6 +108,24 @@ func (fake *fakeIPSetVersioner) GetVersion() (string, error) {
 	return fake.version, fake.err
 }
 
+// New returns a new FakeSysctl
+func NewFakeSysctl() *FakeSysctl {
+	return &FakeSysctl{}
+}
+
+type FakeSysctl struct {
```

Comment:
Is this `FakeSysctl` real used in our test function? Or just for `NewProxier()` input?

Reply:
just for NewProxier() input